### ISSUE TITLE
[telegraf] input port not exposed in Service

### ISSUE THREAD

**Issue by aureq** [influxdata/helm-charts#164]
Hi

Telegraf supports ``[[inputs.syslog]]`` however, the plugin port is not exposed in the telegraf service during deployment.

The current generated output:
```yaml
# Source: telegraf/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: telegraf
  labels:
    helm.sh/chart: telegraf-1.7.21
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: telegraf
    app.kubernetes.io/instance: telegraf
spec:
  type: ClusterIP
  ports:
  - port: 8888
    targetPort: 8888
    name: "health"
  selector:
    app.kubernetes.io/name: telegraf
    app.kubernetes.io/instance: telegraf
```

If the syslog input plugin is enabled, then it would be good to have

Desired output
```yaml
# Source: telegraf/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: telegraf
  labels:
    helm.sh/chart: telegraf-1.7.21
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: telegraf
    app.kubernetes.io/instance: telegraf
spec:
  type: ClusterIP
  ports:
  - port: 8888
    targetPort: 8888
    name: "health"
  - port: 514
    targetPort: 514
    protocol: UDP
    name: "syslog"
  selector:
    app.kubernetes.io/name: telegraf
    app.kubernetes.io/instance: telegraf
```

Thanks.

**Comment by russorat** [comment]
@aureq thanks! would this be a fix you'd like to contribute?

**Comment by aureq** [comment]
Hi @russorat 

Happy to try that, however I have limited bandwidth right now.

**Comment by alespour** [comment]
Closed via #401
